### ISSUE TITLE
PonyDebugger is not logging WkWebView network requests

### ISSUE THREAD

**Issue by paragmajum** [square/PonyDebugger#175]
Is this a known issue related to the fact that WkWebView does not give access to it's networking layer?

**Comment by orta** [comment]
Yeah, WKWebView runs out of process, it's not possible for the App to see those requests at all.

**Comment by asmallteapot** [comment]
TODO: document that we can't view `WKWebView` requests
